src/psyclone/transformations.py

```diff
@@ -2283,9 +2283,14 @@ def apply(self, sched, options=None):
                 current = current.parent
             posn = sched.children.index(current)
 
+        # handle async option
+        if options == None:
```

Comment:
Please document the 'async_queue' option in the docstring. Please also make it clear that if it is set then the same value must also be supplied to any subsequent kernel transformation to ensure synchronisation.